traits/tests/test_ctraits.py

```diff
@@ -303,7 +331,6 @@ class TestCTraitNotifiers(unittest.TestCase):
     """ Test calling trait notifiers and object notifiers. """
 
     def test_notifiers_empty(self):
-
```

Comment:
Please could you revert this spacing change and the similar one on line 316: I think they're unrelated to this PR. (FWIW, they come originally from black, and we might as well keep black's preferred style unless there's a good reason not to.)